cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/MultivariateAnomalyDetection.scala

```diff
@@ -192,14 +197,34 @@ trait MADBase extends HasOutputCol
   val startTime = new Param[String](this, "startTime", "A required field, start time" +
     " of data to be used for detection/generating multivariate anomaly detection model, should be date-time.")
 
-  def setStartTime(v: String): this.type = set(startTime, v)
+  def setStartTime(v: String): this.type = {
```

Comment:
nit: code duplication in both setters, (and elsewhere I think) consider a helper

Reply:
Thanks! Will refactor this part.